### PR TITLE
Add metrics helper functions

### PR DESCRIPTION
**Pull request by Merrickneo**
Changes:
1. Add metrics helper functions
2. Update transactions to refer to the right tables and fields

### CODE REVIEW

File: metrics/database_state.csv
Comment (gabyang) [inline metrics/database_state.csv:1]: what is this file for

File: setup/bash-scripts/test-run/driver-main.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+#     with open(filename, 'r') as file:
+#       for line in file:
+#         process_transaction(line.split(','))
+#   except FileNotFoundError:
+#     print(f"Error: File '{filename}' not found.")
+#   except Exception as e:
+#     print(f"An error occurred: {str(e)}")
+# else:
+#   print("Please provide a filename as an argument.")
+#   sys.exit(1)
```
Comment (gabyang) [inline setup/bash-scripts/test-run/driver-main.py]: remove this

File: transactions.py [hunk 2]
```diff
@@ -14,6 +14,12 @@ def __init__(self):
         port = int(os.getenv("DATABASE_PORT", "5432"))
         password = os.getenv("DATABASE_PASSWORD")
 
+        # host="localhost"
+        # database = "postgres"
+        # username = "postgres"
+        # port = 5432
+        # password = None
+
```
Comment (gabyang) [inline transactions.py:22]: remove this

File: transactionsV3_copy.py [hunk 3]
```diff
@@ -2,19 +2,25 @@
...
+        # database = os.getenv("DATABASE_NAME")
+        # username = os.getenv("DATABASE_USERNAME")
+        # port = int(os.getenv("DATABASE_PORT", "5432"))
+        # password = os.getenv("DATABASE_PASSWORD")
+
+        host="localhost"
+        database = "postgres"
+        username = "postgres"
+        port = 5432
+        password = None
```
Comment (gabyang) [inline transactionsV3_copy.py]: dont commit this

File: transactions_copy.py [hunk 4]
```diff
@@ -11,13 +11,19 @@ class Transactions:
...
+        # database = os.getenv("DATABASE_NAME")
+        # username = os.getenv("DATABASE_USERNAME")
+        # port = int(os.getenv("DATABASE_PORT", "5432"))
+        # password = os.getenv("DATABASE_PASSWORD")
+
+        host="localhost"
+        database = "postgres"
+        username = "postgres"
+        port = 5432
+        password = None
```
Comment (gabyang) [inline transactions_copy.py]: dont commit this

File: metrics/database_state.csv
Comment (Merrickneo) [reply metrics/database_state.csv:1]: Just an output file for the metrics. Just for yall to see the format.

File: setup/bash-scripts/test-run/driver-main.py (hunk 1 above)
Comment (Merrickneo) [reply setup/bash-scripts/test-run/driver-main.py]: Sure

File: transactions.py (hunk 2 above)
Comment (Merrickneo) [reply transactions.py:22]: Easier to test locally!

Comment (Merrickneo) [reply transactions.py:22]: Nvm i can just use a `.env` file

**Comment by Merrickneo** [comment]
@gabyang merge merge

**Pull request merged**